.github/workflows/data-check/checks.yaml

```diff
@@ -0,0 +1,43 @@
+---
+
+- name: Check if {{ table_id }} table exists
+  error: {{ table_id }} does not exist 
+  query: |
+    SELECT "{{ table_id }}" NOT IN (
+        SELECT `table_name`
+        FROM `basedosdados-dev.{{ dataset_id }}.INFORMATION_SCHEMA.TABLES`
+    ) AS failure
+
+- name: Check if select query in {{ table_id }} works
+  error: {{ table_id }} select query not working
+  query: |
+    SELECT NOT EXISTS (
+        SELECT * 
+        FROM `basedosdados-dev.{{ dataset_id }}.{{ table_id }}`
+    ) as failure
+
+{% for column in columns %}
+{% if column.is_in_staging %}
+- name: Check if {{ column.name }} column is null
+  error: {{column.name}} column is null
+  query: |
+    SELECT NOT EXISTS (
+        SELECT `{{ column.name }}`
+        FROM `basedosdados-dev.{{ dataset_id }}.{{ table_id }}`
+        WHERE `{{ column.name }}` IS NOT NULL
+        LIMIT 1
+    ) AS failure
+{% endif %}
+{% endfor %}
+
+{% for primary_key in primary_keys %}
+- name: Check if {{ primary_key }} has unique values
```

Comment:
Essa query deve evitar o loop:

```
SELECT 
  CASE WHEN COUNT(DISTINCT(id_municipio)) / COUNT(*) = 1 THEN 'Pass' ELSE 'Fail' END id_municipio,
  CASE WHEN COUNT(DISTINCT(id_municipio_6)) / COUNT(*) = 1 THEN 'Pass' ELSE 'Fail' END id_municipio_6,
FROM `basedosdados.br_bd_diretorios_brasil.municipio` t
```

Reply:
da pra colocar o `for` no nível do `SELECT` ai ele gera a query inteira e roda uma vez só